pkg/api/converterfromapi.go

```diff
@@ -383,6 +383,17 @@ func convertOrchestratorProfileToVLabs(api *OrchestratorProfile, o *vlabs.Orches
 	if api.OrchestratorVersion != "" {
 		o.OrchestratorVersion = vlabs.OrchestratorVersion(api.OrchestratorVersion)
 	}
+
+	if api.KubernetesConfig != nil {
+		o.KubernetesConfig = &vlabs.KubernetesConfig{}
+		convertKubernetesConfigToVLabs(api.KubernetesConfig, o.KubernetesConfig)
```

Comment:
This is from #473, I'm just re-adding the same code that was previously here.

KubernetesConfig could be nil if the input apimodel to acs-engine does not have a KubernetesConfig section. But I think it is also a good defensive practice to check for nil before accessing pointers.

Is there a better way to write this? What did you have in mind?